README.md

```diff
@@ -192,3 +192,23 @@ python3 -m hearpreprocess.sampler <taskname>
 
 **_NOTE_** : `--mode small` is used to run the task on a
 small version of the dataset for development.
+
+### Breaking change for hear-eval
+
+If the open tasks have changed enough to break the downstream CI,
+(for example in the heareval repo), the [Preprocessed Downsampled HEAR Open
+Tasks](https://github.com/neuralaudio/hear2021-open-tasks-downsampled/tree/main/preprocessed)
+should be updated. 
+
+The version should be bumped up in `hearpreprocess/__init__.py` and the pipeline should
+be run for the open tasks with `--mode small` flag
+
+Thereafter, the following command can be used to copy the tarred files produced by running the pipeline for the open tasks to the repo( Please clone the repo )
+
+```
+cp hear-{VERSION}-speech_commands-v0.0.2-small-44100.tar.gz ./hear2021-open-tasks-downsampled/preprocessed/
+```
+
+The `LATEST` version should also be pushed
+
+An Example of a very obvious breaking changes can be modification of the configuration of the open task.
```

Comment:
Capitalization "An Example"

Also, get rid of word "very" and avoid in scientific writing, as well as other inspecific adverbs like "quite"